pkg/image/api/v1/types.go

```diff
@@ -240,6 +240,11 @@ type ImageStreamStatus struct {
 	// DockerImageRepository represents the effective location this stream may be accessed at.
 	// May be empty until the server determines where the repository is located
 	DockerImageRepository string `json:"dockerImageRepository" protobuf:"bytes,1,opt,name=dockerImageRepository"`
+	// PublicDockerImageRepository represents the public location from where the image can
+	// be pulled outside the cluster. The public URL is constructed by reading the route created for
```

Comment:
This part of the comment is not something an API user cares about.  Remove these two sentences and replace with "this field may be empty if the administrator has not exposed the integrated registry externally".

I think the field name is correct.